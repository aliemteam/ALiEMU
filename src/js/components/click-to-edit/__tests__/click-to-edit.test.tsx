import React from 'react';
import renderer from 'react-test-renderer';

import ClickToEdit from '../index';

it('should render correctly', () => {
    const tree = renderer.create(
        <>
            <ClickToEdit onSave={jest.fn()} />
            <ClickToEdit onSave={jest.fn()} placeholder="With Placeholder" />
        </>,
    );
    expect(tree).toMatchSnapshot();
});
