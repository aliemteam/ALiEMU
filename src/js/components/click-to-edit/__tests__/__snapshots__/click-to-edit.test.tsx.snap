// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
Array [
  <a
    className="btn"
    onClick={[Function]}
    role="button"
    tabIndex={0}
  >
    
  </a>,
  <a
    className="btn"
    onClick={[Function]}
    role="button"
    tabIndex={0}
  >
    With Placeholder
  </a>,
]
`;
